src/walk-manifest.js

```diff
@@ -114,7 +114,15 @@ var walkPlaylist = function(decrypt, basedir, uri, parent, manifestIndex) {
 		parent.content = new Buffer(parent.content.toString().replace(uri, path.relative(path.dirname(parent.file), manifest.file)));
 	}
 
-  manifest.content = syncRequest('GET', manifest.uri).getBody();
+  res = syncRequest('GET', manifest.uri);
+
+  // Don't throw an error if we get a non 200 just return an emtpy array.
```

Comment:
why is this needed? If an error is being returned couldn't this just be handled by a catch block in the calling code?